components/workflow-mgt/org.wso2.carbon.identity.workflow.impl/src/main/java/org/wso2/carbon/identity/workflow/impl/WorkflowImplServiceImpl.java

```diff
@@ -171,13 +163,13 @@ public void deleteHumanTask(WorkflowRequest workflowRequest) throws WorkflowImpl
         List<WorkflowImplServiceListener> workflowListenerList =
                 WorkflowImplServiceDataHolder.getInstance().getWorkflowListenerList();
 
-        try {
+
             for (WorkflowImplServiceListener workflowListener : workflowListenerList) {
                 workflowListener.doPreDeleteHumanTask(workflowRequest);
             }
             int tenantId = CarbonContext.getThreadLocalCarbonContext().getTenantId();
             List<BPSProfile> bpsProfiles = bpsProfileDAO.listBPSProfiles(tenantId);
-            HumanTaskClientAPIAdminStub stub = null;
+        HumanTaskClientAPIAdminClient client = null;
```

Comment:
This has already fixed with https://github.com/wso2/carbon-identity/commit/0c9ac18459c977c3d3f50fcde9c5183a9af69f63